peer/peer.go

```diff
@@ -162,13 +163,9 @@ func Encode(id ID) string {
 
 // FromCid converts a CID to a peer ID, if possible.
 func FromCid(c cid.Cid) (ID, error) {
-	ty := c.Type()
-	if ty != cid.Libp2pKey {
-		s := cid.CodecToStr[ty]
-		if s == "" {
-			s = fmt.Sprintf("[unknown multicodec %d]", ty)
-		}
-		return "", fmt.Errorf("can't convert CID of type %s to a peer ID", s)
+	code := mc.Code(c.Type())
+	if code != mc.Libp2pKey {
+		return "", fmt.Errorf("can't convert CID of type %s to a peer ID", code.String())
```

Comment:
```suggestion
		return "", fmt.Errorf("can't convert CID of type %q to a peer ID", code)
```

fmt's %s will already call the String method by default :) plus, using %q is a bit better so we don't print something confusing if e.g. a code string has a space.